src/store/middleware/inputMiddleware.js

```diff
@@ -42,6 +42,14 @@ function updateDrive(prevComputedInput, computedInput, dispatch) {
       dispatch(requestDrive({ straight, steer }));
     }
   }
+  if (computedInput.drive.activeSuspension) {
+    const { activeSuspension: prevActiveSuspension } = prevComputedInput.drive;
+    const { activeSuspension } = computedInput.drive;
+    if (activeSuspension != prevActiveSuspension) {
+      dispatch(requestJointPower({ activeSuspension,
+        power: computedInput.drive.activeSuspension }));
```

Comment:
You can read more [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer), but what this is doing is creating an object with the keys `"activeSuspension"` and `"power"`, but we want the object to have the keys `"jointName"` and `"power"`.

I assume this was copy-pasted from elsewhere in the code, so it's an easy mistake to make but it's a good habit to review the code you're copying to make sure you're using it correctly.

Anyway, this should be:
```javascript
dispatch(requestJointPower({"jointName": "activeSuspension",
    power: computedInput.drive.activeSuspension }));
```